client/src/pages/submitSecret/form.js

```diff
@@ -37,11 +39,27 @@ const Form = ({formResponse}) => {
     allow_insecure_passphrase: false
   }));
 
+  // HTML replacement
+  const INPUT_PASS_DEFAULT = "<input type=\"password\" id=\"passphrase\" name=\"passphrase\" class=\"form-control\">"
+  const INPUT_PASS_PUB = "<textarea rows=\"7\" cols=\"80\" id=\"passphrase\" name=\"passphrase\" placeholder=\"-----BEGIN PUBLIC KEY-----&#10;MIIBojANBgkqhkiG9w0BAQEFAAOCAY8AMIIBigKCAYEAvk3&#10;...&#10;-----END PUBLIC KEY-----\" onChange={handleInputChange} className=\"form-control\" rows=\"3\"></textarea>"
```

Comment:
Style. Inconsistent use of single and double quotes. Use single quotes

Reply:
Removed now, fixed